### PR TITLE
Dropped dbname variable and set QUOTED_IDENTIFIER to ON

### PR DESCRIPTION
**Pull request by yitam**
To prepare these scripts for future development purpose:
1. Modify the scripts to SET QUOTED_IDENTIFIER to 'ON' -- run sqlcmd with -I and bcp with -q
1. Also drop 'Use DbName' or the use of dbname variable

### CODE REVIEW

File: test/functional/setup/setup_dbs.py [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
 import argparse
-from subprocess import Popen, PIPE
 from exec_sql_scripts import *
 
 def setupTestDatabase(conn_options, dbname, azure):
     sqlFiles = ['test_types.sql', '168256.sql', 'cd_info.sql', 'tracks.sql']
-    
-    # for Azure, must specify the database for the sql scripts to work
-    if (azure.lower() == 'yes'):
-        conn_options += ' -d ' + dbname
```
Comment (david-puglielli) [inline test/functional/setup/setup_dbs.py]: Why is this not needed anymore?

File: test/functional/setup/168256.sql [hunk 2]
```diff
@@ -1,6 +1,3 @@
-USE $(dbname) 
-GO
-
```
Comment (david-puglielli) [inline test/functional/setup/168256.sql]: All of these changes are to make the scripts work with DW - but then we are using these scripts for all the other platforms we test on, correct? I assume we've tested these scripts in Bamboo?

Comment (yitam) [reply test/functional/setup/168256.sql]: With this change we no longer need the clumsy workaround outside Windows platforms

File: test/functional/setup/exec_sql_scripts.py [hunk 3]
```diff
@@ -15,29 +12,17 @@ def executeCommmand(inst_command):
...
 
 def executeSQLscript(sqlfile, conn_options, dbname):
-    if platform.system() == 'Windows':
-        executeSQLscriptWindows(sqlfile, conn_options, dbname)
-    elif platform.system() == 'Linux' or platform.system() == 'Darwin':
-        executeSQLscriptUnix(sqlfile, conn_options, dbname)
-    
-def executeSQLscriptWindows(sqlfile, conn_options, dbname):
-    inst_command  = 'sqlcmd ' + conn_options + ' -i ' + sqlfile + ' -v dbname =' + dbname
+    inst_command  = 'sqlcmd -I ' + conn_options + ' -i ' + sqlfile + ' -d ' + dbname
```
Comment (yitam) [inline test/functional/setup/exec_sql_scripts.py]: The variable dbname is always appended at the end of `sqlcmd`

File: test/functional/setup/setup_dbs.py (hunk 1 above)
Comment (yitam) [reply test/functional/setup/setup_dbs.py]: Please see my other comment

**Pull request merged**